Documentation/design/installconfig.md

```diff
@@ -58,7 +58,6 @@ type Cluster struct {
 ```go
 // AWS converts AWS related config.
 type AWS struct {
-    AutoScalingGroupExtraTags []map[string]string `json:"tectonic_autoscaling_group_extra_tags,omitempty" yaml:"autoScalingGroupExtraTags,omitempty"`
```

Comment:
This is previous work of tectonic-installer. No need to edit this.

Reply:
> This is previous work of tectonic-installer. No need to edit this.

Ah, thanks.  Dropped with c9351ed -> 46de8e5.